### PR TITLE
Lock chromedriver version to 74 and enable ruby tests

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
Ruby tests didn't run because of the new version of the chromedriver

# Solution
Download chromedriver version 74 to enable Ruby tests

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by bartzet** [comment]
> I can see that the ruby tests produce no output. Are they running?

Yes, they are as you can see the result of the passed scenarios at the end of the job. We don't see the scenarios while they are running as at some point we decided to reduce the logs on the CI.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://693-pr-onfido-sdk-ui-onfido.surge.sh

File: test/js/utils/misc.js [hunk 1]
```diff
@@ -23,4 +23,4 @@ export const spawnPrinter = (color, stdout, stderrorPrefix) => process => {
   });
 }
 
-export const SHELL_COLOR_BLUE = "\x1b[34m"
+export const SHELL_COLOR_BLUE = "\x1b[32m"
```
Comment (kopijunkie) [inline test/js/utils/misc.js]: This export constant name doesn't match up with the import name in `main.js` file above (`SHELL_COLOR_GREEN`)

Comment (bartzet) [reply test/js/utils/misc.js]: Didn't save the file before pushing 🤦‍♂ done now.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-693-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://693-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**